okapi/services/caches/reports/status/docs.xml

```diff
@@ -0,0 +1,40 @@
+<xml>
+    <brief>Query the status of geocache reports</brief>
+    <issue-id>TODO</issue-id>
+    <desc>
+        <p>This method queries the status of one or more geocache reports that were
+        submitted trough <a href="submit.html">services/caches/reports/submit</a>.</p>
+        <!-- TODO: link the terms of use here. -->
+    </desc>
+    <req name='report_uuids'>
+        <p>Pipe-separated list of report IDs.</p>
+    </req>
+    <common-format-params/>
+    <returns>
+        <p>An array, each element corresponding to one of the provided reports IDs and
+        having one of these values:</p>
+        <ul>
+            <li>
+                <b>null</b> - there is no report with this ID, or
+            </li>
+            <li>
+                <p>a dictionary, having this structure:</p>
+                <ul>
+                    <li>
+                        <p><b>status</b> - the status of the report, which is
+                        either <b>Unassigned</b>, <b>In progress</b> or <b>Closed</b>,</p>
+                    </li>
```

Comment:
Are you sure there won't be any more statuses in the future? If there will be, a new field will be introduced, yes?

Reply:
Actually, there is also the status "Look here!" at OCPL, which cannot unambiguously be mapped to "Unassigned" or "In progress".  To keep things simple, I have now merged all three to "Open".

I think it's very unlikely that a new status will be added which cannot be mapped to either "Open" or "Closed". If so, we would indeed need a status2 field.